files/default/hopsworks_templates/airflow_dag.py

```diff
@@ -60,6 +60,7 @@
 					 project_name="${operator.projectName}",
 					 task_id="${operator.id}",
 					 job_name="${operator.jobName}",
+                     job_arguments="${operator.jobArgs}",
```

Comment:
Nitpick, can you adjust the indentation to match the other arguments?

Reply:
Fixed